spec/integration/lhm_spec.rb

```diff
@@ -296,7 +296,7 @@
         end
       end
 
-      assert_match "Required triggers do not exist", exception.message
+      assert_match "Verification failed, aborting early", exception.message
```

Comment:
Because we remove the triggers before Chunker runs, we're failing because of `verifier` before we can fail in the atomic rename step. This seems okay to me, but I just wanted to explain this change to an unrelated test.